packages/viewer/readme.md

```diff
@@ -43,6 +43,28 @@ file that references our published JS bundles from a CDN.
 </html>
 ```
 
+If you want to interact with the web component via JavaScript, you'll need to ensure the browser has registered the custom elements prior to use. 
+
+One option is to call this prior to any method calls on the viewer:
+```js
+  await window.customElements.whenDefined('vertex-viewer');
+```
```

Comment:
What do you think of just documenting `defineCustomElements`? It looks to be the documented/recommended approach in Stencil.

Reply:
I'm ok with this. I do think it will be weird if we don't document it though, but then we use it in our sample index.html file for local dev.